scheduler/src/cook/rest/api.clj

```diff
@@ -1973,22 +2001,23 @@
           group-txns (map #(make-group-txn % (get group-uuid->job-dbids
                                                   (:uuid %)
                                                   []))
-                          groups)
-          pool-name (pool/pool-name-or-default (:pool/name pool))]
+                          groups)]
 
       (let [user (get-in ctx [:request :authorization/user])]
         (rate-limit/spend! rate-limit/job-submission-rate-limiter user (count jobs))
```

Comment:
If we have jobs empty and as well as job-pool-name-maps not empty, then this would be wrong..

Reply:
`::jobs` is now gone, see above